lighthouse-core/gather/gatherers/js-usage.js

```diff
@@ -27,9 +27,19 @@ class JsUsage extends Gatherer {
     const driver = passContext.driver;
 
     const coverageResponse = await driver.sendCommand('Profiler.takePreciseCoverage');
+    const scriptUsages = coverageResponse.result;
     await driver.sendCommand('Profiler.stopPreciseCoverage');
     await driver.sendCommand('Profiler.disable');
-    return coverageResponse.result;
+
+    /** @type {Record<string, Array<LH.Crdp.Profiler.ScriptCoverage>>} */
+    const usageByUrl = {};
```

Comment:
this seemed better here.

maybe a perf benefit too, since computed artifacts operate on an array of ScriptCoverages and it's faster to take advantage of referential equality than deep nesting that would occur if the arrays were different (from each audit that would have to group coverages by url themselves)